CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Fixed
 
 - `DataGrid`: the `componentWillReceiveProps` lifecycle is replaced, as it will be decrecated in react v17 ([@LennertBlommaert](https://github.com/LennertBlommaert) in [#331](https://github.com/teamleadercrm/ui/pull/331))
+- `Input`: unwanted double borders on input fields with connected buttons ([@driesd](https://github.com/driesd) in [#334](https://github.com/teamleadercrm/ui/pull/334))
```

Comment:
shouldn't this be

'removed unwanted double borders ....' not sure what the actual action was right now?